src/tabs/BluetoothListPage.js

```diff
@@ -21,21 +21,14 @@ import {
   VSeparator,
   ListSeparator,
   ListCell,
-} from './components/cell';
-import {BTList, BluetoothEnableButton} from './components/BTList';
+} from '../components/cell';
+import {BTList, BluetoothEnableButton} from '../components/BTList';
 
-import {BTConnection} from './bt_communication/bt_connection';
+import {BTConnection} from '../bt_communication/bt_connection';
 
 export const BluetoothListPage = () => {
-  const [
-    BTInit,
-    BTRemove,
-    EnableBT,
-    DisableBT,
-    BTLogin,
-    lista,
-    bolEnableBlu,
-  ] = BTConnection();
+  const [BTInit, BTRemove, EnableBT, DisableBT, BTLogin, lista, bolEnableBlu] =
+    BTConnection();
```

Comment:
Então, eu ia fazer a recomendação de transformar esse BTConnection pra que exportasse um objeto ao invés de um vetor. O problema de exportar vetores é que você é obrigado a puxar todos os elementos do vetor. Se fosse um objeto, você poderia puxar somente os elementos que você vai utilizar:

```suggestion
  const {BTInit, BTRemove, EnableBT, DisableBT, BTLogin} =
    BTConnection();
```

Mas se for mudar isso tem que mudar no export também